Flaky test test_read_emep_multiple_dirs_hour_error

**Describe the bug**

`test_read_emep_multiple_dirs_hour_error` is occasionally failing in CI

 - Pyaerocom version: main-dev
 - Computing platform: CI
 - Configuration file (if applicable):
<details><summary>Error message:</summary>

```
=================================== FAILURES ===================================
___________________ test_read_emep_multiple_dirs_hour_error ____________________

tmp_path = PosixPath('/tmp/pytest-of-runner/pytest-0/test_read_emep_multiple_dirs_h0')

    def test_read_emep_multiple_dirs_hour_error(tmp_path: Path):
        data_path = emep_data_path(tmp_path, "hour", vars_and_units={"prmm": "mm"})
        reader = ReadMscwCtm(data_dir=str(data_path))
>       with pytest.raises(ValueError) as e:
E       Failed: DID NOT RAISE <class 'ValueError'>

/home/runner/work/pyaerocom/pyaerocom/tests/io/mscw_ctm/test_reader.py:505: Failed```

</details>

--- a/pyaerocom/io/mscw_ctm/reader.py
+++ b/pyaerocom/io/mscw_ctm/reader.py
@@ -574,6 +574,8 @@ class ReadMscwCtm(GriddedReader):
                 start_date = min([x for x in [start_date, file_start_date] if x is not None])
                 end_date = max([x for x in [file_end_date, file_end_date] if x is not None])
 
+            logger.info(f"******* Found enddate, start_date: {end_date}, {start_date}")
+            logger.info(f"{(end_date - start_date) / np.timedelta64(1, 'h')}")
             if (end_date - start_date) / np.timedelta64(1, "h") > (366 * 24):
                 raise ValueError(
                     f"ts_type {ts_type} can not be hourly when using multiple years ({start_date} - {end_date})"

--- a/pyaerocom/io/mscw_ctm/reader.py
+++ b/pyaerocom/io/mscw_ctm/reader.py
@@ -574,8 +574,6 @@ class ReadMscwCtm(GriddedReader):
                 start_date = min([x for x in [start_date, file_start_date] if x is not None])
                 end_date = max([x for x in [end_date, file_end_date] if x is not None])
 
-            logger.info(f"******* Found enddate, start_date: {end_date}, {start_date}")
-            logger.info(f"{(end_date - start_date) / np.timedelta64(1, 'h')}")
             if (end_date - start_date) / np.timedelta64(1, "h") > (366 * 24):
                 raise ValueError(
                     f"ts_type {ts_type} can not be hourly when using multiple years ({start_date} - {end_date})"

--- a/pyaerocom/io/mscw_ctm/reader.py
+++ b/pyaerocom/io/mscw_ctm/reader.py
@@ -572,7 +572,7 @@ class ReadMscwCtm(GriddedReader):
                     file_end_date = nc["time"][:].data.max()
 
                 start_date = min([x for x in [start_date, file_start_date] if x is not None])
-                end_date = max([x for x in [file_end_date, file_end_date] if x is not None])
+                end_date = max([x for x in [end_date, file_end_date] if x is not None])
 
             logger.info(f"******* Found enddate, start_date: {end_date}, {start_date}")
             logger.info(f"{(end_date - start_date) / np.timedelta64(1, 'h')}")

